src/components/shared/CallNodeContextMenu.js

```diff
@@ -507,9 +495,7 @@ class CallNodeContextMenu extends PureComponent<Props> {
         onShow={this._onShow}
         onHide={this._onHide}
       >
-        {this.props.isContextMenuVisible
-          ? this.renderContextMenuContents()
-          : null}
```

Comment:
No need to have the conditional here, as this `render` method will return early with `null` when the context menu is not visible (that is, when `callNodeIndex === null`).